Modules/_threadmodule.c

```diff
@@ -1499,50 +1676,50 @@ unhandled exception; a stack trace will be printed unless the exception\n\
 is SystemExit.\n");
 
 static PyObject *
-thread_PyThread_start_joinable_thread(PyObject *module, PyObject *func)
+thread_PyThread_start_joinable_thread(PyObject *module, PyObject *fargs,
+                                      PyObject *fkwargs)
 {
+    static char *keywords[] = {"function", "daemon", NULL};
+    PyObject *func = NULL;
+    int daemon = 0;
```

Comment:
Should the default be `daemon = 1` to preserve existing behavior? I think that threads created by calling `start_joinable_thread()` directly did not previously delay shutdown.